update: iOS Overview for iOS 26 (#3130)
privacyguides/privacyguides.org: docs/os/ios-overview.md

Reviewer: @jordan-warne and @friadev, let me know what you think about the simplification of these sections to reduce the (slight) maintenance burden, since both of you discussed this in the team chat.

Reviewer 2: That's a good way of doing it 👍 that also means it's aligned with the other section too.

#### Apple Intelligence & Siri

This is available if your device supports **[Apple Intelligence](https://support.apple.com/guide/iphone/apple-intelligence-and-privacy-iphe3f499e0e/ios)**. Apple Intelligence uses a combination of on-device processing and their **[Private Cloud Compute](https://security.apple.com/blog/private-cloud-compute)** for things that take more processing power than your device can provide.

To see a report of all the requests made to Apple's servers, you can navigate to **Privacy & Security** → **Apple Intelligence Report** and press **Export Activity** to see activity from the either the last 15 minutes or 7 days, depending on what you set it for. Similar to the **App Privacy Report** which shows you the recent permissions accessed by the apps on your phone, the Apple Intelligence Report likewise shows what is being sent to Apple's servers while using Apple Intelligence.

By default, Apple Intelligence integrates with ChatGPT. If you don't want ChatGPT integration, you can navigate to **ChatGPT**:

- [ ] Turn off **Use ChatGPT**

You can also have it ask for confirmation every time if you leave ChatGPT integration on:

> - [x] Turn on **Confirm Requests** 

Reviewer 3: My settings look like this:

<img width="1206" height="2622" alt="image" src="https://github.com/user-attachments/assets/6fec6d8d-c7f9-457d-8748-0cf47684126f" />

Author: I guess it’s different if you already enabled it, that’s frustrating.

If you wish to use Face ID or Touch ID, you can go ahead and set it up now. Your phone will use the password you set up earlier as a fallback in case your biometric verification fails. Biometric unlock methods are primarily a convenience, although they do stop surveillance cameras or people over your shoulder from watching you input your passcode.

If you use biometrics, you should know how to turn them off quickly in an emergency. Holding down the [side button](https://support.apple.com/en-us/105103) and *either* volume button until you see the Slide to Power Off slider will disable biometrics, requiring your passcode to unlock. Your passcode will be required after your device restarts.

You can similarly disable biometrics by pressing the side button five times, or for devices with Touch ID, you can hold down the side button and nothing else. Make sure you try this in advance, so you know which method works for your device.

> **Stolen Device Protection** adds additional security intended to protect your personal data if your device is stolen while unlocked. If you use biometrics, we recommend enabling this protection:

Reviewer 2: > It’s just called Find My though isn’t it

Maybe it would be better to be more specific, I think in this case it would be Find My iPhone, just because "Stolen Device Protection" is only available on iPhones not iPads (yet)

The [color of the message bubble](https://support.apple.com/en-us/104972) in the Messages app indicates whether your messages are E2EE or not. A blue bubble indicates that you're using iMessage with E2EE, while a green bubble indicates the other party is using either the outdated SMS and MMS protocols or RCS. RCS on iOS is **not** E2EE. Currently, the only way to have E2EE in Messages is for both parties to be using iMessage on Apple devices.

If either you or your messaging partner have iCloud Backup enabled without Advanced Data Protection, the encryption key will be stored on Apple's servers, meaning they can access your messages.

> By default, you trust Apple's identity servers that you're messaging the right person. To eliminate the need to trust Apple's servers, you can enable **[Contact Key Verification](https://support.apple.com/en-us/118246)**. At the top of the **Settings** app where your name is, select it, then go to **Contact Key Verification**.

Reviewer 3: ```suggestion
By default, you trust Apple's identity servers that you're messaging the right person. To defend yourself from a potentially malicious server, you can enable **[Contact Key Verification](https://support.apple.com/en-us/118246)**. At the top of the **Settings** app where your name is, select it, then go to **Contact Key Verification**.
```

- [x] Turn on **Verification in iMessage**

> Both you and your contacts need to enable Contact Key Verification and follow Apple's [instructions](https://support.apple.com/en-us/118246#verify) for the security assurances mentioned above to take effect.

Reviewer 3: ```suggestion
When this option is enabled, you will see a Verification option in the contacts page of an iMessage conversation. Next time you are physically near the person you're messaging, click "Verify Contact..." to begin the verification process. Once verified, this feature ensures that if your or your contact's encryption keys are changed in the future, you will be notified.

Both you and your contacts need to enable Contact Key Verification and follow Apple's [instructions](https://support.apple.com/en-us/118246#verify) for the security assurances mentioned above to take effect.

A notice that the verification keys have changed is not necessarily an indication of compromise. For example, your contact could have changed devices or reset their iCloud Keychain. However, it is noteworthy and you should avoid sharing sensitive information until you can re-verify that contact in-person again.
```

Reviewer 2: I think we should link to Apple's guide in case it changes, perhaps it would be better to do a bit of an explanation of why you would want to verify a contact on iMessage in the first place.

If you are notified that your contact's encryption key has changed, there could be the possibility that their device has be compromised, you should exercise caution until you can verify their key again in-person.

The [color of the message bubble](https://support.apple.com/en-us/104972) in the Messages app indicates whether your messages are E2EE or not. A blue bubble indicates that you're using iMessage with E2EE, while a green bubble indicates the other party is using either the outdated SMS and MMS protocols or RCS. RCS on iOS is **not** E2EE. Currently, the only way to have E2EE in Messages is for both parties to be using iMessage on Apple devices.

If either you or your messaging partner have iCloud Backup enabled without Advanced Data Protection, the encryption key will be stored on Apple's servers, meaning they can access your messages.

By default, you trust Apple's identity servers that you're messaging the right person. Enable **[Contact Key Verification](https://support.apple.com/en-us/118246)** to combat this. At the top of the settings screen where your name is, select it. Navigate to **Contact Key Verification**.

- [x] Turn on **Verification in iMessage**

> Follow Apple's [instructions](https://support.apple.com/en-us/118246#verify) to verify your contacts (they also need to enable Contact Key Verification).

Reviewer: ```suggestion
Both you and your contacts need to enable Contact Key Verification and follow Apple's [instructions](https://support.apple.com/en-us/118246#verify) for the security assurances mentioned above to take effect.
```

You can hide an app by long-pressing on it and selecting **Require Face ID/Touch ID** → **Hide and Require Face ID/Touch ID**. Note that pre-installed Apple apps, as well as the default web browser and email app, cannot be hidden. Hidden apps reside in a **Hidden** folder at the bottom of the App Library, which can be unlocked using biometrics. This folder appears in the App Library whether you hid any apps or not, which provides you a degree of plausible deniability.

### Guided Access

Sometimes you might want to hand your phone to someone to make a call or do a specific task, but you don't want them to have full access to your phone. In these cases, you can quickly enable **[Guided Access](https://support.apple.com/guide/iphone/lock-iphone-to-one-app-iph7fad0d10/ios)** to lock the phone to one specific app until you authenticate.

<div class="admonition warning" markdown>
<p class="admonition-title">Warning</p>

> Guided Access isn't foolproof, it's possible you could leak data you don't intend to or it could be bypassed.

Reviewer: ```suggestion
Guided Access is not foolproof, as it can't protect against you unknowingly leaking your data or against bypasses of the feature itself.
```

Author: Yeah basically when you’re handing your phone to someone to use.

</div>

### Redacting Elements in Images

If you need to hide information in a photo, you can use Apple's built-in editing tools to do so.

> If your device supports it, you can use the [Clean Up](https://support.apple.com/en-us/121429) feature to pixelate faces or remove objects from images.

Reviewer: ```suggestion
You can use the [Clean Up](https://support.apple.com/en-us/121429) feature on supported devices to pixelate faces or remove objects from images.
```

- Open the **Photos** app and tap the photo you have selected for redaction
- Tap the :material-tune:
- Tap the button labeled **Clean Up**
- Draw a circle around whatever you want to redact. Faces will be pixelated, and it will attempt to delete anything else.

Our warning [against blurring text](../data-redaction.md) also applies here, so we recommend to instead add a black shape with 100% opacity over it. In addition to redacting text, you can also black out any face or object using the **Photos** app.

- Tap the image you have selected for redaction

> - Tap the :material-tune: → markup symbol → plus icon

Reviewer: ```suggestion
- Tap the :material-tune: → :material-dots-horizontal: → Markup → :material-plus:
```

### Before First Unlock

If your threat model includes [:material-target-account: Targeted Attacks](../basics/common-threats.md#attacks-against-specific-individuals){ .pg-red } that involve forensic tools, and you want to minimize the chance of exploits being used to access your phone, you should restart your device frequently. The state *after* a reboot but *before* unlocking your device is referred to as "Before First Unlock" (BFU), and when your device is in that state it makes it [significantly more difficult](https://belkasoft.com/checkm8_glossary) for forensic tools to exploit vulnerabilities to access your data. This BFU state allows you to receive notifications for calls, texts, and alarms, but most of the data on your device is still encrypted and inaccessible. This can be impractical, so consider whether these trade-offs make sense for your situation.

> iPhones automatically [reboot](https://support.apple.com/guide/security/protecting-user-data-in-the-face-of-attack-secf5549a4f5/1/web/1#:~:text=On%20an%20iPhone%20or%20iPad%20with%20iOS%2018%20and%20iPadOS%2018%20or%20later%2C%20a%20new%20security%20protection%20will%20restart%20devices%20if%20they%20remain%20locked%20for%20a%20prolonged%20period%20of%20time.) over a period of time if they're not unlocked.

Reviewer: ```suggestion
iPhones [automatically reboot](https://support.apple.com/guide/security/protecting-user-data-in-the-face-of-attack-secf5549a4f5/1/web/1#:~:text=On%20an%20iPhone%20or%20iPad%20with%20iOS%2018%20and%20iPadOS%2018%20or%20later%2C%20a%20new%20security%20protection%20will%20restart%20devices%20if%20they%20remain%20locked%20for%20a%20prolonged%20period%20of%20time.) if they're not unlocked after a period of time.
```

### MTE

> iPhone 17 and later offer a security enhancement called [Memory Tagging Extension](https://developer.arm.com/documentation/108035/0100/Introduction-to-the-Memory-Tagging-Extension) (MTE) which helps prevent memory bugs and makes exploiting the device significantly harder. This protection is always on, but it's dependent on hardware so it's not available to older devices. You can read more about it on their [blog post](https://security.apple.com/blog/memory-integrity-enforcement/).

Reviewer: ```suggestion
The iPhone 17 line and later offer a security enhancement called [Memory Tagging Extension](https://developer.arm.com/documentation/108035/0100/Introduction-to-the-Memory-Tagging-Extension) (MTE), which makes a device significantly harder to exploit by prevent memory corruption bugs from occurring. This always-on protection depends on hardware support, so it's not available for older devices.

For more details on Apple's implementation of MTE, read the [blog post](https://security.apple.com/blog/memory-integrity-enforcement) published by Apple Security Research.
```
